README.md

```diff
@@ -1,57 +1,78 @@
 
-- npm install [package name]
-- mvn install
-- ...
+If you're using **WSL** on Windows with **Ubuntu 20.04**, you can follow the steps below to get Swift working. For all other OS's and additional setup steps, please refer to the official [Swift documentation](https://www.swift.org/).
 
-### Quickstart
-(Add steps to get up and running quickly)
+```bash
+# Install Swift on Ubuntu 20.04 LTS x64
+# Ewerton Scaboro da Silva, 2022-02-14
 
-1. git clone [repository clone url]
-2. cd [respository name]
-3. ...
+apt-get update
+apt-get install curl clang libicu-dev git libatomic1 libicu66 libxml2 libcurl4 zlib1g-dev libbsd0 tzdata libssl-dev libsqlite3-dev libblocksruntime-dev libncurses5-dev libdispatch-dev -y
+mkdir ~/swift
+cd ~swift
+wget https://download.swift.org/development/ubuntu2004/swift-DEVELOPMENT-SNAPSHOT-2022-02-03-a/swift-DEVELOPMENT-SNAPSHOT-2022-02-03-a-ubuntu20.04.tar.gz
+tar -xvzf ./swift-DEVELOPMENT-SNAPSHOT-2022-02-03-a-ubuntu20.04.tar.gz -C ~/swift
+echo 'export PATH="~/swift/swift-DEVELOPMENT-SNAPSHOT-2022-02-03-a-ubuntu20.04/usr/bin:$PATH"' >> ~/.profile
+echo 'export PATH="~/swift/swift-DEVELOPMENT-SNAPSHOT-2022-02-03-a-ubuntu20.04/usr/bin:$PATH"' >> ~/.bashrc
+export PATH="~/swift/swift-DEVELOPMENT-SNAPSHOT-2022-02-03-a-ubuntu20.04/usr/bin:$PATH" 
+
+```
+
+## Getting Started
 
+Once you have the prerequisites completed:
 
-## Demo
+- Clone the sample repo
+- Configure the `Sources/demo/config.swift` file with your credentials
+- Add your certificates to the `certs` folder (`client.pem` and `client-key.pem`)
+- Build and run the demo:
 
-A demo app is included to show how to use the project.
+```swift
+swift package clean
+swift build
+swift run demo
+```
 
-To run the demo, follow these steps:
+## Known Issues and Limitations
 
-(Add steps to start up the demo)
+- Support for MQTT over WebSockets: Swift MQTT (used in this sample) does not support Web Sockets. [CocoaMQTT](https://github.com/emqx/CocoaMQTT) could be an alternative, since it leverages [StarScream](https://github.com/daltoniam/Starscream). [MQTT-nio](https://github.com/adam-fowler/mqtt-nio) might be an alternative as well. 
+- This sample leverages Swift NIO (for network) and Swift NIO-SSL (for TLS) are supported by Apple. Swift MQTT client is a 3rd party library, not created nor supported by Apple.
+- No SAS Token utilization is implemented on this sample.
+- No reconnection and no retries have been implemented. They need to be implemented by the client application.
 
-1.
-2.
-3.
+## Support
 
-## Resources
+The Swift sample is an open source solution, it is **NOT** a Microsoft supported solution or product. For bugs and issues with the codebase please log an issue in this repo.
 
-(Any additional resources or related projects)
+## Contributing
 
-- Link to supporting information
-- Link to similar sample
-- ...
+If you would like to contribute to the Swift sample, please base your own branch and pull request (PR) off our dev branch.
```

Comment:
Is this needed? OR did you copy from one of our other repos?

Reply:
Pulled from another. A lot of this is from Wellington